lib/util/error.js

```diff
@@ -650,6 +650,11 @@ shaka.util.Error.Code = {
    */
   'INCONSISTENT_DRM_ACROSS_PERIODS': 4038,
 
+  /**
+   * The HLS manifest requires undeclared variables.
```

Comment:
How about "refers to" instead of "requires".  I think that might be clearer.

It might also be good to add the variable name to the error using the "data" array.  Just add additional parameters to the `new shaka.util.Error()` call and document them here.

Reply:
I have added that when the error is launched also include the missing varibale.